app/kumactl/cmd/install/testdata/install-cp-helm/securityContext.golden.yaml

```diff
@@ -455,6 +455,8 @@ spec:
             requests:
               cpu: 100m
               memory: 256Mi
+            limits:
+              memory: 256Mi
```

Comment:
Might worth increasing this limit no?

Reply:
Apparently [it's best practice to set limit = request](https://home.robusta.dev/blog/kubernetes-memory-limit) & [@thockin](https://web.archive.org/web/20220802181951/https://twitter.com/thockin/status/1134193838841401345)

Reply:
Yeah it just feels like 256Mi will get us many folks complaining of CPs OOMing no?

Reply:
Perhaps, if we're regularly using that much memory.

Reply:
I only modified the cpu request/limit checker as suggested in the linked issue in the description to not enforce a CPU limit.